screenshooter.go

```diff
@@ -57,7 +81,7 @@ func (ss *Screenshooter) Take(postURL url.URL, element string, filename string)
 		return err
 	}
 
-	err = ioutil.WriteFile(fmt.Sprintf("%s/%s.png", ss.SavePath, filename), buf, 0644)
+	err = ioutil.WriteFile(fmt.Sprintf("%s/%s.png", ss.savePath, filename), buf, 0644)
```

Comment:
If `ss.savePath` path is empty then the file would written to `/filename` which is possibly not desirable.

Reply:
> If `ss.savePath` path is empty then the file would written to `/filename` which is possibly not desirable.

Uhm yeah, I might want to have some sort of default and/or error handling here. Will do.

Reply:
> not sure how much you care, you could use https://golang.org/pkg/path/filepath/ to support cross platform

Not a lot, to be honest, at least not for now. I didn't know that pkg existed, so I've learnt something today, thanks! :smile: